See https://redmine.gromacs.org/projects/gromacs/repository/revisions/9baf5dcc540878dc0b15d8b945ac760d36fbf75d/diff
IBM VSX should be allowed with GCC 9.x
Patch added to EasyBuild by Simon Branford, University of Birmingham
--- cmake/gmxManageSimd.cmake.orig	2021-02-15 09:23:01.644332000 +0000
+++ cmake/gmxManageSimd.cmake	2021-02-15 09:23:56.364277044 +0000
@@ -266,7 +266,7 @@
 
     # IBM_VSX and gcc > 9 do not work together, so we need to prevent people from
     # choosing a combination that might fail. Issue #3380.
-    if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER "9")
+    if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 10)
         message(FATAL_ERROR "IBM_VSX does not work together with gcc > 9. Disable SIMD support (slower), or use an older version of the GNU compiler")
     endif()
 
